pkg/cloud/aws/actuators/cluster/actuator.go

```diff
@@ -28,31 +32,36 @@ import (
 	clusterv1 "sigs.k8s.io/cluster-api/pkg/apis/cluster/v1alpha1"
 	client "sigs.k8s.io/cluster-api/pkg/client/clientset_generated/clientset/typed/cluster/v1alpha1"
 	controllerError "sigs.k8s.io/cluster-api/pkg/controller/error"
+	"sigs.k8s.io/cluster-api/pkg/controller/remote"
 )
 
 //+kubebuilder:rbac:groups=awsprovider.k8s.io,resources=awsclusterproviderconfigs;awsclusterproviderstatuses,verbs=get;list;watch;create;update;patch;delete
 //+kubebuilder:rbac:groups=cluster.k8s.io,resources=clusters;clusters/status,verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups=,resources=secrets,verbs=create;update
```

Comment:
I'd do get & create and omit update

Reply:
Would we need the update long term? I added in case we need it at some point, so it's easier to upgrade clusters running v1alpha1, wdyt?

Reply:
I'd say to avoid update for now, if we need it in the future we can add it then.

Reply:
If we need to "update", the user can manually delete the secret, and it will be recreated on the next reconcile pass.